src/com/missafternoon/budjen/Application.java

```diff
@@ -20,11 +25,31 @@ public Application(final TransactionData data, final Logger logger) {
         this.logger = logger;
     }
 
-    public void start() {
+    public void start(String[] args) {
+        String arg;
+        if (args.length == 0) {
```

Comment:
This if block is redundant. `arg` is already null. You really want to check with `!= null` I think.

Explicitly assigning something (using `=`) is generally a code smell, i.e. a sign that something's sub-optimal.